dist/ansible/scripts/ovn-setup.sh

```diff
@@ -66,6 +67,7 @@ wait_for_condition is_api_server_ready
 #    networkPluginName: cni
 #  serviceNetworkCIDR: 172.30.0.0/16
 
+# ocp 4.0 permits multiple CIDRs that include the subnet length
```

Comment:
@pecameron it's not OCP4 that does this, it's ovn-kubernetes itself since Jacob added that support.